scripts/eval/eval-prompt-change.py

```
@@ -0,0 +1,561 @@
+        Path(args.output).write_text(json_output, encoding="utf-8")
+        print(f"\n  Results written to {args.output}", file=sys.stderr)
+    else:
+        print(json_output)
+
+    # Summary
+    print(f"\n{'='*60}", file=sys.stderr)
+    print(f"  ACCEPTANCE GATE: {gate['verdict']}", file=sys.stderr)
+    print(f"{'='*60}", file=sys.stderr)
+    print(f"  Before: {gate['before_score']:.0%}  After: {gate['after_score']:.0%}  "
+          f"Delta: {gate['delta']:+.0%}", file=sys.stderr)
+    print(f"  Criteria:", file=sys.stderr)
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)